docs/smartContractExamples/LetterOfCredit

```diff
@@ -0,0 +1,20 @@
+# Аккредитив
+# Транзакция предоставляет MultiSignature
+
+contract (signature: MultiSig, transaction: Transaction) = {
+  def isValidSig(signature: MultiSig, message: Array[Byte], key: Array[Byte]): Bool = {
+    true in signature.map(lamb (x: Array[Byte]) = checkSig(x, message, key))
```

Comment:
You'd probably better use `allOf(signature.map(lamb (x: Array[Byte]) = checkSig(x, message, key)))`